Sources/SignHereLibrary/Services/iTunesConnectService.swift

```diff
@@ -361,7 +367,9 @@ internal class iTunesConnectServiceImp: iTunesConnectService {
         let profileName: String = "\(certificateId)_\(profileType)_\(clock.now().timeIntervalSince1970)"
         var devices: CreateProfileRequest.CreateProfileRequestData.Relationships.Devices? = nil
         // ME: App Store profiles cannot use UDIDs
-        if !["IOS_APP_STORE", "MAC_APP_STORE", "TVOS_APP_STORE", "MAC_CATALYST_APP_STORE"].contains(profileType) {
+        if Store.allCases
```

Comment:
If we don't already have an exhaustive test over this implementation, it would be great to add one. Let's also make sure any existing test cases capture the implicit `else` condition.